src/radio-button.jsx

```diff
@@ -78,7 +77,7 @@ const RadioButton = React.createClass({
     inputStyle: React.PropTypes.object,
 
     /**
-     * Used internally by `RadioButtonGroup`. Use the `labelPosition` property of `RadioButtonGroup` instead.
+     * @ignore
      */
     /* Where the label will be placed next to the radio button. */
```

Comment:
You could move this into the comment block above @igonore. 

I think I may have said to remove the "used internally" comment, but in hindsight it might be instructive to leave it to explain why the prop is `@ignore`d...